app-commons/src/main/scala/org/bitcoins/commons/jsonmodels/dlc/DLCMessage.scala

```diff
@@ -178,9 +178,9 @@ object DLCMessage {
         fundingInputs.map { input =>
           val obj = mutable.LinkedHashMap(
             "prevTx" -> Str(input.prevTx.hex),
-            "prevTxVout" -> Num(input.prevTxVout.toInt),
-            "sequence" -> Num(input.sequence.toInt),
-            "maxWitnessLength" -> Num(input.maxWitnessLen.toInt)
+            "prevTxVout" -> Num(input.prevTxVout.toLong.toDouble),
```

Comment:
Wait so why is this needed?

Reply:
`toInt` wasn't big enough to cover for all `UInt32` and and `Num()` can't take a long, so had to a double